src/test/scala/beam/agentsim/agents/household/FastHouseholdCAVSchedulingSpec.scala

```diff
@@ -214,7 +214,16 @@ class FastHouseholdCAVSchedulingSpec
     P2.addPlan(plan2)
     population.addPerson(P2)
 
-    household.setMemberIds(JavaConverters.bufferAsJavaList(mutable.Buffer(P1.getId, P2.getId)))
+    val P3: Person = population.getFactory.createPerson(Id.createPersonId(household.getId + "_P3"))
```

Comment:
How does this test the empty plan? Tests act as a means of documentation also, so it would be preferred to use better variable names (although I see that hasn't been the case - so not really a blocker, more opinion). More important is there is no explicit validation - ie assertDoesNotThrow - I would even take a comment, but reverting back to variable naming - you could change plan3 to emptyPlan and then P3 to personWithEmptyPlan.